README.md

```diff
@@ -121,32 +129,90 @@ integrations:
           token: ${GITLAB_TOKEN}
 ```
 
-5. Add proxy config:
+**Note:** You can have more than one GitLab instance.
 
-```yaml
-'/gitlabci':
-    target: '${GITLAB_URL}/api/v4'
-    allowedMethods: ['GET']
-    headers:
-        PRIVATE-TOKEN: '${GITLAB_TOKEN}'
+5. Add the GitLab Filler Processor:
+
+`packages/backend/src/plugins/catalog.ts`
+
+```ts
+// packages/backend/src/plugins/catalog.ts
+import { GitlabFillerProcessor } from '@immobiliarelabs/backstage-plugin-gitlab-backend';
+
+export default async function createPlugin(
+    env: PluginEnvironment
+): Promise<Router> {
+    const builder = await CatalogBuilder.create(env);
+    //...
+    // Add this line
+    builder.addProcessor(new GitlabFillerProcessor(env.config));
+    //...
+    const { processingEngine, router } = await builder.build();
+    await processingEngine.start();
+    return router;
+}
+```
+
+This allows auto-filling of the annotations.
```

Comment:
Please add  "like the project id and slug" and it's good to me! Also, move this under `5. Add the GitLab Filler Processor:` so it's immediately clear what it does!

Reply:
Done!